codegen/src/main/resources/macros/marshaller/rest/UriMemberMarshallerMacro.ftl

```diff
@@ -5,7 +5,7 @@
     String uriResourcePath = "${resourcePath}";
 
     <#if resourcePath?contains("?")>
-        uriResourcePath = software.amazon.awssdk.util.UriResourcePathUtils.addStaticQueryParamtersToRequest(request, uriResourcePath);
```

Comment:
![I trusted you](http://s2.quickmeme.com/img/5e/5e20b8d890a8cdc928c842242bd79bab14f818b7b0761ed19207e7ca8dcfc369.jpg)

Reply:
I'm sorry that I let you down @spfink